src/hdf5/cloudsat_netcdf_to_ioda.py

```diff
@@ -0,0 +1,199 @@
+        ds_g.variables['ReflectivityAttenuated'][:] = tmpMat.copy()
+        zvar.setncattr_string("units", str("dBz"))
+
+        g = "GsiHofXBc"
+        ds_g = ds.createGroup(g)
+        zvar = ds_g.createVariable('ReflectivityAttenuated', np.float32, refl_dims, fill_value=fill_value, zlib=True)
+        ds_g.variables['ReflectivityAttenuated'][:] = tmpMat.copy()
+        zvar.setncattr_string("units", str("dBz"))
+
+        g = "GsiObsBias"
+        ds_g = ds.createGroup(g)
+        zvar = ds_g.createVariable('ReflectivityAttenuated', np.float32, refl_dims, fill_value=fill_value, zlib=True)
+        ds_g.variables['ReflectivityAttenuated'][:] = tmpMat.copy()
+        zvar.setncattr_string("units", str("dBz"))
+
+        # we don't need the channel element anymore
+        cpr_obs = cpr_obs.isel(channel=0)
+
+        g = 'MetaData'
+        ds_g = ds.createGroup(g)
+
+        unixtime = cpr_obs.epoch_time.values
+
+        varid = ds_g.createVariable('dateTime', np.int64, ('Location',))
+        ds_g.variables['dateTime'][:] = unixtime.astype(np.int64)
+        ds_g.variables['dateTime'].setncattr('units', 'seconds since 1970-01-01T00:00:00Z')
+
+        ds_g.createVariable('sensorCentralFrequency', np.float32, ('Channel',), fill_value=fill_value)
+        ds_g.createVariable('sensorCentralWavenumber', np.float32, ('Channel',), fill_value=fill_value)
+        ds_g.createVariable('gsi_use_flag', np.int32, ('Channel',), fill_value=-2147483647)
+        ds_g.createVariable('sensorPolarizationDirection', np.int32, ('Channel',), fill_value=-2147483647)
+        ds_g.createVariable('latitude', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('longitude', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('sensorScanPosition', np.int32, ('Location',), fill_value=-2147483647)
+        ds_g.createVariable('sensorChannelNumber', np.int32, ('Channel',), fill_value=-2147483647)
+        ds_g.createVariable('sensorViewAngle', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('sensorZenithAngle', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('sensorAzimuthAngle', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('solarAzimuthAngle', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('solarZenithAngle', np.float32, ('Location',), fill_value=fill_value)
+        # ds_g.createVariable('time', np.float32, ('Location',), fill_value=fill_value)
+        ds_g.createVariable('sequenceNumber', np.int32, ('Location',), fill_value=-2147483647)
+
+        ds_g.variables['sensorCentralFrequency'][:] = np.array([94.05]).astype(np.float32)    # cpr_sim.Frequency.values.astype(np.float32)
+        ds_g.variables['sensorCentralWavenumber'][:] = np.array([3.1371]).astype(np.float32)  # cpr_sim.Wavenumber.values.astype(np.float32)
```

Comment:
@imoradi these lines make me nervous is this correct

I will look into it some more in a bit